StitchCore-iOS/StitchCore_iOSTests/Core/TestUtil/StitchIntegrationTestCase.swift

```diff
@@ -32,4 +32,28 @@ internal class StitchIntegrationTestCase: XCTestCase {
         wait(for: [exp], timeout: defaultTimeoutSeconds)
     }
 
+    public func registerAndLogin(email: String = email,
```

Comment:
[general] Who is going to volunteer to change how we are doing testing to closely match Java with Cuckoo and such?

Reply:
Once Jason is done with his big iOS ticket I'll try to tackle it. I gotta get better at writing tests since it definitely slows me down I think learning how to use a mocking framework will be a good exercise.